nmostesting/IS04Utils.py

```diff
@@ -12,10 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from . import TestHelper
 
 from .NMOSUtils import NMOSUtils
 
 
 class IS04Utils(NMOSUtils):
     def __init__(self, url):
         NMOSUtils.__init__(self, url)
+
+    def get_node(self):
+        """Get node self resource from the Node API"""
+
+        valid_resource, resource = TestHelper.do_request("GET", self.url + "self")
+        if valid_resource and resource.status_code == 200:
+            return resource.json()
+
```

Comment:
Does flake8 warn if you put an explicit `return None` in (for the error case)?

Reply:
Do you mean something like this?

```
        valid_resource, resource = TestHelper.do_request("GET", self.url + "self")
        if valid_resource and resource.status_code == 200:
            return resource.json()
        else:
            return None
```

Reply:
Yes.

Reply:
Ok yep. WIll do